src/app/pages/send/choose-crypto-asset/components/choose-crypto-asset.layout.tsx

```diff
@@ -10,6 +10,8 @@ export function ChooseCryptoAssetLayout({ children }: StackProps) {
       flexGrow={1}
       flexDirection="column"
       justifyContent="start"
+      maxHeight={['unset', '85vh']}
+      overflowY="scroll"
```

Comment:
This can be changed for sure if wrong, this is what I was seeing in fullscreen...

![Screen Shot 2023-05-31 at 4 28 07 PM](https://github.com/hirosystems/wallet/assets/6493321/51451609-41c4-4bb0-8164-f0b04a23e81b)